en/docs/developer-portal/manage-subscription.md

```diff
@@ -0,0 +1,20 @@
+# Subscribe to an API
+
+A subscription to an API makes it possible for the application to to know which APIs it can authenticate with it's keys. You have to subscribe the **published** API  to an application to invoke it using the application credentials.
```

Comment:
```suggestion
A subscription to an API makes it possible for the application to know which APIs it can authenticate with its keys. You have to subscribe the **published** API to an application to invoke it using the application credentials.
```